### ISSUE TITLE
Missing postcss loader in .bootstraprc?

### ISSUE THREAD

**Issue by lazzrek** [shakacode/bootstrap-loader#40]
Is the postcss loader not required in the styleLoaders field to prefix the bootstrap 4 sass?

**Comment by justin808** [comment]
@lazzrek We have the examples in the /examples directory.

Can you be more specific with what doesn't work and can you more clearly show what is the issue?

**Comment by lazzrek** [comment]
I tried to use the [Card Columns](http://v4-alpha.getbootstrap.com/components/card/#columns) bt4 component. But it didn't work because the css was not prefixed. 

Snippet from [/examples/css-modules/webpack.dev.config.js](https://github.com/shakacode/bootstrap-loader/blob/master/examples/css-modules/webpack.dev.config.js#L43):
```javascript
{
    test: /\.scss$/,
    loaders: [
      'style',
      'css?modules&importLoaders=2&localIdentName=[name]__[local]__[hash:base64:5]',
      'postcss',
      'sass',
    ],
}
```
Snippet from [/examples/css-modules/.bootstraprc](https://github.com/shakacode/bootstrap-loader/blob/master/examples/css-modules/.bootstraprc#L11):
```yaml
# Webpack loaders, order matters
styleLoaders:
  - style
  - css
  - sass
```
I fixed the prefixing by adding the postcss loader to the `styleLoaders` list:
```yaml
# Webpack loaders, order matters
styleLoaders:
  - style
  - css
  - postcss
  - sass
```
Is this the correct way to fix my problem?

**Comment by RazkoSA** [comment]
When I add .bootstraprc with customization, postcss functionality does not work.   Without bootstraprc I can use bootstrap4 as mentioned in angular2 webpack example.   You have not directly answered post css or have not put in proper example of using postcss in .bootstraprc for bootstrap 4.

**Comment by justin808** [comment]
@RazkoSA @lazzrek it will be a while before we move our project to Bootstrap 4. We won't have time to fully test this until we do. In the meantime, if you can provide help in debugging this, we'd be very grateful, and we'd be happy to help you get in some PRs to fix this.

Possibly Bootstrap 4 has evolved since we first got this working.

CC: @alexfedoseev

**Comment by justin808** [comment]
@RazkoSA @lazzrek Can you fork the project and provide an example for the /examples directory showing the issue?

**Comment by justin808** [comment]
Fixed in #56.
